hyper/client/aws/ec2.go

```diff
@@ -730,8 +759,20 @@ func StartServer(manifestPath string, remoteCfg HyperConfig.EC2RemoteConfigurati
 		},
 	}
 
+	version := "0.0.2"
```

Comment:
If you want to test this before the PR gets merged, you will need to change this to `version := "0.0.1-remotestart.0.83f73bf"` since the 0.0.2 binary won't exist until this is merged in